packages/n/nushell/package.yml

```diff
@@ -1,22 +1,23 @@
-name       : nushell
```

Comment:
The formatting of this file is messed up. Please re-format it like it was previously (check other `package.yml` files if you need a visual reference.

Reply:
Apologies, emacs has some auto formatter on yaml files I have not figured out how to stop or the tool its using to format, I usually edit in fundamental mode to fix it but easy to forget.

I will fix it now and update